src/raftstore/store/worker/pd.rs

```diff
@@ -325,7 +324,7 @@ impl<T: PdClient> Runner<T> {
             is_hb_receiver_scheduled: false,
             region_peers: HashMap::default(),
             store_stat: StoreStat::default(),
-            start_ts: time_now_sec(),
+            start_ts: TimeStamp::now_sec(),
```

Comment:
I don't think this is a correct refactor. 

The ts used in transaction is a defined as a monotonically increasing transaction id, which is implemented by using a physical timestamp and logical counter.

The timestamp in this line however, is a real unix timestamp (unix epoch) which is not a transaction id.

Reply:
The function called is the same, it is just named differently. [before](https://github.com/tikv/tikv/pull/5857/files#diff-0763f38a729a7c5243a167eb407c4cdfL40-L45) [after](https://github.com/tikv/tikv/pull/5857/files#diff-f0ae7fe8fe55588d53c4d164f2b2c467R226-R232)

Reply:
I know it's the same. However you made two mistakes.

Let's simply give the structure `TimeStamp` a more accurate name `TransactionId`, then you can see the problem:

Mistake 1. The return value of time_now_sec is a unix epoch, but not a thing that is compatible with `TransactionId` because of missing the logical part.

Mistake 2. The PD client needs a unix epoch, instead of a transaction id.

Reply:
I see, thanks. I think this comes back to the question of whether there are two kinds of timestamp. Let me make a proposal in a top-level comment.